### PR TITLE
[Currency] Implement toggleable interface on currencies.

### PR DESCRIPTION
**Pull request by gperdomor**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Implement toggleable interface on currencies.
Fix phpspec directory on currencies component.
The currencies cannot be deleted.

### CODE REVIEW

File: src/Sylius/Component/Currency/Model/Currency.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Currency implements CurrencyInterface
     protected $updatedAt;
 
     /**
-     * Constructor
+     * Constructor.
```
Comment (pamil) [inline src/Sylius/Component/Currency/Model/Currency.php]: It's redundant.

**Comment by pjedrzejewski** [comment]
@gperdomor Requires rebasing and I will merge. :) Thank you @gperdomor!

**Comment by gperdomor** [comment]
@pjedrzejewski refactored :)

**Comment by pjedrzejewski** [comment]
Thanks @gperdomor! :+1:

**Pull request merged**